client/state/selectors/get-next-page-query.js

```diff
@@ -12,6 +12,7 @@ import getCurrentMediaQuery from 'calypso/state/selectors/get-current-media-quer
 import 'calypso/state/media/init';
 
 const DEFAULT_QUERY = Object.freeze( { number: 20 } );
+const isString = ( arg ) => typeof arg === 'string';
```

Comment:
Can you rewrite the condition that uses the function? I.e.,
```js
[ 'string', 'number' ].includes( typeof pageHandle )
```

Reply:
Neat idea, I'll take it. Thanks 👍

Reply:
Addressed in 03bb1764da1dcfcfba965cbb4d5d8987d1c11984